README.md

```diff
@@ -1,2 +1,19 @@
-# powerflows-dmn-model
+# About Power Flows DMN Model
 Power Flows DMN Model
+
+# How to contribute to the repository
+Contributors wishing to join Power Flows project have to comply with a few rules: 
+
+* A preferred IDE is IntelliJ IDEA with default formatting styles;
+* Committed source code must be high quality;
+* Committed source code must be formatted;
+* A commit has to solve an issue from the product backlog. If the issue is missing, please add one before creating the pull request. Refer the issue in commit summary;
+* Unit tests coverage should be at 80% or higher;
+* In order to start contribution, a contributor has to fork the project;
+* Once an issue is fixed, a contributor has to create a commit to his forked repository;
+* In order to merge changes to the project repository, contributor has to create a pull request with single commit.  
+
```

Comment:
no source encoding and EOL requirements ?

Reply:
Good idea, added.